internal/sync/sync.go

```diff
@@ -19,6 +20,9 @@ import (
 const (
 	DepbotApiKey     = "DEPBOT_API_KEY"
 	DepbotServerAddr = "DEPBOT_SERVER_ADDR"
+
+	syncFlagApiKey       = "--api-key"
```

Comment:
I don't see the reason for having this variable at this level. Its only used once.